src/MSBuild/OutOfProcTaskHostNode.cs

```diff
@@ -821,14 +821,8 @@ private void RunTask(object state)
                 SetTaskHostEnvironment(taskConfiguration.BuildProcessEnvironment);
 
                 // Set culture
-#if FEATURE_CULTUREINFO_SETTERS
-                CultureInfo.CurrentCulture = taskConfiguration.Culture;
-                CultureInfo.CurrentUICulture = taskConfiguration.UICulture;
-
-#else
                 Thread.CurrentThread.CurrentCulture = taskConfiguration.Culture;
                 Thread.CurrentThread.CurrentUICulture = taskConfiguration.UICulture;
-#endif
```

Comment:
This is interesting. I originally flipped it like the others, but it failed compilation in the 3.5 task host. Looking back through time at the original commit:
https://github.com/Microsoft/msbuild/blob/82177a50da735cc0443ac10fa490d69368403d71/src/XMakeCommandLine/OutOfProcTaskHostNode.cs#L768-L769

This is actually what we had for Dev14. I could do `#IF CLR2`, but it seems like this is as good?

https://referencesource.microsoft.com/#mscorlib/system/globalization/cultureinfo.cs,1f5907384ac6bb55

Thoughts?